### PR TITLE
Fix coding style for NewModuleManifestCommand

### PR DESCRIPTION
**Pull request by pougetat**
The purpose of this PR is to fix some of the coding style in NewModuleManifestCommand

- Comments for members
- 3 member names have been changed so that the use of nameof can be consistent throughout the file : nameof(Modules.CLRVersion) => nameof(CLRVersion). Unfortunately this means not following certain conventions for these 3 member names. 

## PR Checklist

- [x] [PR has a meaningful title](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
    - Use the present tense and imperative mood when describing your changes
- [x] [Summarized changes](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
- [x] [Change is not breaking](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#making-breaking-changes)
- [x] [Make sure all `.h`, `.cpp`, `.cs`, `.ps1` and `.psm1` files have the correct copyright header](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
- [x] This PR is ready to merge and is not [Work in Progress](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---work-in-progress).
    - If the PR is work in progress, please add the prefix `WIP:` or `[ WIP ]` to the beginning of the title (the `WIP` bot will keep its status check at `Pending` while the prefix is present) and remove the prefix when the PR is ready.
- **User-facing changes**
    - [x] Not Applicable
    - **OR**
    - [ ] [Documentation needed](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
        - [ ] Issue filed: 
- **Testing - New and feature**
    - [ ] N/A or can only be tested interactively
    - **OR**
    - [x] [Make sure you've added a new test if existing tests do not effectively test the code changed](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#before-submitting)
        - [ ] [Add `[feature]` to your commit messages if the change is significant or affects feature tests](https://github.com/PowerShell/PowerShell/blob/master/docs/testing-guidelines/testing-guidelines.md#requesting-additional-tests-for-a-pr)

### CODE REVIEW

File: src/System.Management.Automation/engine/Modules/NewModuleManifestCommand.cs [hunk 1]
```diff
@@ -58,10 +58,10 @@ public object[] NestedModules
         private object[] _nestedModules;
 
         /// <summary>
-        /// Set the GUID in the manifest file.
+        /// Gets or sets the GUID in the manifest file.
         /// </summary>
         [Parameter]
-        public Guid Guid
+        public Guid GUID
```
Comment (iSazonov) [inline src/System.Management.Automation/engine/Modules/NewModuleManifestCommand.cs]: Please revert - we should use camel style. Also it is public API (breaking change).

File: src/System.Management.Automation/engine/Modules/NewModuleManifestCommand.cs [hunk 2]
```diff
@@ -516,12 +516,12 @@ public object PrivateData
...
 
         /// <summary>
-        /// Specify the HelpInfo URI.
+        /// Gets or sets the HelpInfo URI.
         /// </summary>
         [Parameter]
         [AllowNull]
         [SuppressMessage("Microsoft.Design", "CA1056:UriPropertiesShouldNotBeStrings")]
-        public string HelpInfoUri
+        public string HelpInfoURI
```
Comment (iSazonov) [inline src/System.Management.Automation/engine/Modules/NewModuleManifestCommand.cs]: Please revert - it is public API (breaking change).

File: src/System.Management.Automation/engine/Modules/NewModuleManifestCommand.cs [hunk 3]
```diff
@@ -58,10 +58,10 @@ public object[] NestedModules
...
 
         /// <summary>
-        /// Set the GUID in the manifest file.
+        /// Gets or sets the GUID in the manifest file.
         /// </summary>
         [Parameter]
-        public Guid Guid
+        public Guid GUID
         {
             get { return _guid; }
```
Comment (iSazonov) [inline src/System.Management.Automation/engine/Modules/NewModuleManifestCommand.cs]: @pougetat You could review all such unneeded private fields and remove them. Better in follow PR.

File: src/System.Management.Automation/engine/Modules/NewModuleManifestCommand.cs (hunk 1 above)
Comment (pougetat) [reply src/System.Management.Automation/engine/Modules/NewModuleManifestCommand.cs]: done

File: src/System.Management.Automation/engine/Modules/NewModuleManifestCommand.cs (hunk 2 above)
Comment (pougetat) [reply src/System.Management.Automation/engine/Modules/NewModuleManifestCommand.cs]: done

File: src/System.Management.Automation/engine/Modules/NewModuleManifestCommand.cs (hunk 3 above)
Comment (pougetat) [reply src/System.Management.Automation/engine/Modules/NewModuleManifestCommand.cs]: sounds good

**Review by iSazonov — APPROVED** [review]

**Pull request merged**